nemo_text_processing/fst_alignment/alignment.py

```diff
@@ -99,7 +99,8 @@ def parse_args():
 WHITE_SPACE = "\u23B5"
 ITN_MODE = "itn"
 TN_MODE = "tn"
-tn_itn_symbols = list(string.ascii_letters + string.digits) + list("$\:+-=")
```

Comment:
? What was going on here?

Reply:
`list("$\:+-=")` implicitly converts to a list with each element being a character. Since we needed to properly escape the backslash, though, I thought it might be clearer to make that list explicit, since otherwise it'd be a bit confusing -- `list("$\\:+-=")` and `list("$\:+-=")` do identical things, but the former relies on folks having deeper knowledge of escape characters.